lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.py

```diff
@@ -0,0 +1,26 @@
+import maya.api.OpenMaya as om
+from pxr import Sdf
+import mayaUsd
+import ufe
+
+# Adapted from testMayaUsdSchemasMayaReference.py.
+def createMayaReferencePrim(ufePathStr, mayaReferencePath, mayaNamespace):
+    '''Create a Maya reference prim parented to the argument path.'''
+
+    # Append reference name to parent Sdf path.
+    ufePath = ufe.PathString.path(ufePathStr)
+    parentPath = ufePath.segments[1]
```

Comment:
Could you explain the hard-coded [1] here? If I understand it, the code assumes the second segment is a USD path and adds '/MayaReference'  to it? I found the comment is not entirely clear. I think the comment would be more helpful if it said something like:

Extract the USD path segment from the UFE path and adds the MayaReference under it

Or something similar.

Reply:
O.K.